src/sharepoint/rest/Queryable.ts

```diff
@@ -233,4 +249,12 @@ export class QueryableInstance extends Queryable {
         this._query.add("$select", selects.join(","));
         return this;
     }
+    public expand(...expands: string[]): QueryableInstance {
+        this._query.add("$expand", expands.join(","));
+        return this;
+    }
+    public orderby(...orderbys: string[]): QueryableInstance {
```

Comment:
I'm not sure queryable instance should get orderby? Wouldn't that only apply to collections?